tests/integration/sklearn-iris.yaml

```
@@ -0,0 +1,19 @@
+apiVersion: "serving.kserve.io/v1beta1"
+kind: "InferenceService"
+metadata:
+  name: "sklearn-v2-iris"
+spec:
+  predictor:
+    model:
+      modelFormat:
+        name: sklearn
+      protocolVersion: v2
+      runtime: kserve-sklearnserver
+      storageUri: "gs://kfserving-examples/models/sklearn/1.0/model"
```

```suggestion
      storageUri: "gs://kfserving-examples/models/sklearn/2.0/model"
```
Does this has to do with protocol version or the model itself?

---

This is the model itself, check https://kserve.github.io/website/0.12/get_started/first_isvc/#2-create-an-inferenceservice for more info